ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "crypto"
-version = "2.7.2"
+version = "2.7.3"
```

Comment:
IMO we should do a minor version bump (as per semver conventions), since we are adding a new functionality

Reply:
+1